src/main/resources/loot_example.yml

```diff
@@ -7,57 +7,51 @@
 # such as chance, id, amount, damage, rolls, name, level, or enchant.  They should be unique headings though as if
 # they are the same, errors may occur.
 justaname:
-  '1':
+  1: # The number could be anything, it is only used as a section title.
```

Comment:
Why did you end up adding the quotes here and removing the comments?

Reply:
Oof, probably an accident on my part